detection_rules/misc.py

```diff
@@ -49,6 +54,10 @@
 """.strip().format("\n".join(' * ' + line for line in LICENSE_LINES))
 
 
+ROOT_DIR = Path(__file__).parent.parent
+CUSTOM_RULES_DIR = os.getenv('CUSTOM_RULES_DIR', None)
```

Comment:
This essentially means that a user will be able to install the CLI in their python virtual environment and use the rule loader/tests/etc. against rules in a completely different folder right? They don't have to necessarily be within the detection-rules repo right?

Reply:
I'm not sure, but you should test any scenarios you have such as this as part of the review